third-party/eazy/v1/openapi.yaml

```diff
@@ -896,13 +922,29 @@ paths:
           schema:
             type: string
             example: 'GREETING'
+      requestBody:
+        description: The message you would like to update
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                body:
+                  description: Text of the automated message
+                  type: string
+                  example: 'Sorry, we are currently closed. Your question will be answered as soon as possible.'
+                enabled:
+                  description: Is the automated message enabled?
+                  type: boolean
+                  example: false
```

Comment:
Cannot test it. What is the proper fix?

Reply:
Pushed to the master so I guess that I have guessed that right... https://github.com/tyntec/api-collection/pull/106/files#diff-fa1b1ded6b1a42d44b72a651240624ebe14243d63d9999faf2dfac83d70a8ed5R945-R960